tools/slo-generator/tests/unit/fixtures/migrate_format.py

```diff
@@ -0,0 +1,140 @@
+        "cut_after_bucket_number": 31,
+        "good_is_below_cut": False,
+        "filter_valid": "project=\"brunore-gaestd-test\" AND metric.type=\"custom.googleapis.com/datapipeline/throughput\" AND metric.labels.pipeline_category = \"gcs2bq\""
+    }
+}
+
+NEW_1 = {
+    "name": "throughput",
+    "target": 0.9,
+    "service_name": "datapipelines",
+    "feature_name": "gcs2bq",
+    "exporters": [{
+        "class": "PubSub",
+        "project_id": "brunore-stackdriver-test",
+        "topic_name": "slo-achievement"
+    }],
+    "backend": {
+        "class": "Stackdriver",
+        "method": "exponential_distribution_cut",
+        "project_id": "brunore-stackdriver-host-project",
+        "measurement": {
+            "threshold_bucket": 31,
+            "good_below_threshold": False,
+            "filter_valid": "project=\"brunore-gaestd-test\" AND metric.type=\"custom.googleapis.com/datapipeline/throughput\" AND metric.labels.pipeline_category = \"gcs2bq\""
+        }
+    }
+}
+
+OLD_2 = {
+    "service_name": "datapipelines",
+    "feature_name": "teradata2bq",
+    "slo_name": "availability",
+    "slo_target": 0.99,
+    "stackdriver_host_project_id": "brunore-stackdriver-host-project",
+    "slo_achievement_project_id": "brunore-stackdriver-test",
+    "slo_achievement_topic_name": "slo-achievement",
+    "measurement": {
+        "slo_type": "good-bad-ratio",
+        "filter_good": "project=\"brunore-gaestd-test\" AND metric.type=\"custom.googleapis.com/datapipeline/job_count\" AND metric.labels.metric_purpose = \"availability\" AND metric.labels.pipeline_category = \"teradata2bq\" AND metric.labels.event_type = \"good\"",
+        "filter_bad": "project=\"brunore-gaestd-test\" AND metric.type=\"custom.googleapis.com/datapipeline/job_count\" AND metric.labels.metric_purpose = \"availability\" AND metric.labels.pipeline_category = \"teradata2bq\" AND metric.labels.event_type = \"bad\""
+    }
+}
+
+
+NEW_2 = {
+    "name": "availability",
+    "target": 0.99,
+    "service_name": "datapipelines",
+    "feature_name": "teradata2bq",
+    "exporters": [{
+        "class": "PubSub",
+        "project_id": "brunore-stackdriver-test",
+        "topic_name": "slo-achievement"
+    }],
+    "backend": {
+        "class": "Stackdriver",
+        "project_id": "brunore-stackdriver-host-project",
+        "method": "good_bad_ratio",
+        "measurement": {
+            "filter_good": "project=\"brunore-gaestd-test\" AND metric.type=\"custom.googleapis.com/datapipeline/job_count\" AND metric.labels.metric_purpose = \"availability\" AND metric.labels.pipeline_category = \"teradata2bq\" AND metric.labels.event_type = \"good\"",
+            "filter_bad": "project=\"brunore-gaestd-test\" AND metric.type=\"custom.googleapis.com/datapipeline/job_count\" AND metric.labels.metric_purpose = \"availability\" AND metric.labels.pipeline_category = \"teradata2bq\" AND metric.labels.event_type = \"bad\""
+        }
+    }
+}
+
+def deep_eq(d1, d2):
```

Comment:
This sorting and serialization to json is unnecessary overhead as `dict.__eq__` handles the recursion necessary for checking deep equality so `d1 == d2` will yield same result as this function.

some hand spot-checking.
```python3
>>> nd1 = {'a': 1}
>>> nd2 = {'b': 2, 'c': 3}
>>> a = {'one': nd1, 'two': nd2}
>>> b = {'two': nd2, 'one': nd1} # swap order doesn't matter
>>> a == b
True
>>> nd2_a = {'b': 1 + 1, 'c': 3} # just defining same keys and vals to show not nested object specific.
>>> b = {'two': nd2_a, 'one':nd1}
>>> a == b
True
>>> nd2_b = {'b': 0, 'c': 3}
>>> b = {'two': nd2_b, 'one':nd1}
>>> a == b
False
```

Reply:
Sure, this script is only used for one particular case (a customer was using scripts that this package is made of, and this is just a small migration script to help). It's supposed to be run once, so it doesn't really matter if it's not very efficient (although I agree that it could be better).